features/bevragen/persoon/verblijfplaats/adres/protocollering-antwoord.feature

```diff
@@ -0,0 +1,81 @@
+#language: nl
+
+Functionaliteit: protocollering van het antwoord voor verblijfplaats Adres
+
+  Rule: In het antwoord geleverde velden worden geprotocolleerd als de elementnummers volgens Logisch ontwerp BRP
+    Dit is een 6-cijferige code, met zo nodig voorloopnul voor categorieën.
+    Alleen velden die echt geleverd zijn, dus een waarde hebben in de response, worden opgenomen als geleverde rubriek
+    Ook alle velden die worden meegeleverd voor het bepalen van het verblijfplaats type worden opgenomen.
+    
+    Abstract Scenario: Protocolleren van geleverde velden voor <fields veld> voor een persoon met BAG adres met huisletter en toevoeging
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende gegevens
+      | naam            | waarde     |
+      | anummer (01.10) | 8468619643 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                               | waarde   |
+      | gemeente van inschrijving (09.10)  | 0518     |
+      | functie adres (10.10)              | W        |
+      | datum aanvang adreshouding (10.30) | 20190714 |
+      | datum ingang geldigheid (85.10)    | 20190714 |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | naam                                       | waarde           |
+      | gemeente_code                              | 0518             |
+      | straatnaam (11.10)                         | Borgesiusstraat  |
+      | naam openbare ruimte (11.15)               | Borgesiusstraat  |
+      | huisnummer (11.20)                         | 103              |
+      | huisletter (11.30)                         | b                |
+      | huisnummertoevoeging (11.40)               | 2                |
+      | postcode (11.60)                           | 2497BV           |
+      | woonplaats (11.70)                         | Scheveningen     |
+      | identificatiecode verblijfplaats (11.80)   | 0518010000747448 |
+      | identificatiecode nummeraanduiding (11.90) | 0518200000747446 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000012                       |
+      | fields              | <fields>                        |
+      Dan is protocollering opgeslagen met de volgende gegevens
+      | anummer    | request_zoek_rubrieken | request_geleverde_rubrieken |
```

Comment:
Ik mis in dit feature een rule voor 'request_zoek_rubrieken'.

Reply:
De rule op request_zoek_rubrieken staat in protocollering-vraag.feature

Reply:
Ik zie in deze PR geen 'protocollering-vraag.feature' wel een 'features/bevragen/persoon/verblijfplaats/dev/protocollering-vraag-gba.feature'. Bedoel je die soms?

Reply:
Ah, ik zie het nu. Ze zijn allemaal hernoemd naar '......-gba.feature'.